code/modules/spells/spell_types/vampire_powers.dm

```diff
@@ -0,0 +1,416 @@
+		target.reagents.add_reagent(/datum/reagent/medicine/salbutamol, 10) //incase you're choking the victim
+		for(var/progress = 0, progress <= 3, progress++)
+			switch(progress)
+				if(1)
+					to_chat(target, "<span class='warning'>Visions of dread flood your vision...</span>")
+					to_chat(user, "<span class='notice'>We begin to drain [target]'s blood in, so Lilith can bless it.</span>")
+				if(2)
+					to_chat(target, "<span class='danger'>Demonic whispers fill your mind, and they become irressistible...</span>")
+				if(3)
+					to_chat(target, "<span class='danger'>The world blanks out, and you see a demo- no ange- demon- lil- glory- blessing... Lilith.</span>")
+					to_chat(user, "<span class='notice'>Excitement builds up in you as [target] sees the blessing of Lilith.</span>")
+			if(!do_mob(user, target, 70))
+				to_chat(user, "<span class='danger'>The pact has failed! [target] has not became a vampire.</span>")
+				to_chat(target, "<span class='notice'>The visions stop, and you relax.</span>")
+				return
+		if(!QDELETED(user) && !QDELETED(target))
+			to_chat(user, "<span class='notice'>. . .</span>")
+			to_chat(target, "<span class='italics'>Come to me, child.</span>")
+			sleep(10)
+			to_chat(target, "<span class='italics'>The world hasn't treated you well, has it?</span>")
+			sleep(15)
+			to_chat(target, "<span class='italics'>Strike fear into their hearts...</span>")
+			to_chat(user, "<span class='notice italics bold'>They have signed the pact!</span>")
+			to_chat(target, "<span class='userdanger'>You sign Lilith's Pact.</span>")
+			target.mind.store_memory("<B>[user] showed you the glory of Lilith. <I>Respect them and aid in their mission, however you are not required to follow their every order.</I></B>")
+			add_vampire(target)
+			//Make nearly free and work only on the dead because, yknow vampires are undead, and add enthralling for living perhaps with the same spell?
+
+/obj/effect/proc_holder/spell/self/revive
+	name = "Revive"
+	gain_desc = "You have gained the ability to revive after death... However you can still be cremated/gibbed, and you will disintergrate if you're in the chapel!"
+	desc = "Revives you, provided you are not in the chapel!"
+	blood_used = 0
+	stat_allowed = TRUE
+	charge_max = 1000
+	action_icon = 'icons/mob/vampire.dmi'
+	action_icon_state = "coffin"
+	action_background_icon_state = "bg_demon"
+	vamp_req = TRUE
+
+/obj/effect/proc_holder/spell/self/revive/cast(list/targets, mob/user = usr)
+	if(!is_vampire(user) || !isliving(user))
+		revert_cast()
+		return
+	if(user.stat != DEAD)
+		to_chat(user, "<span class='notice'>You aren't dead enough to do that yet!</span>")
+		revert_cast()
+		return
+	if(user.reagents.has_reagent(/datum/reagent/water/holywater))
+		to_chat(user, "<span class='danger'>You cannot revive, holy water is in our system!</span>")
+		return
+	var/mob/living/L = user
+	if(istype(get_area(L.loc), /area/chapel))
```

Comment:
Only if chapel? Why not anyware with blessed tiles?

Reply:
because then you can spray the entire station.
would be nice to make it so they can't rez if it's too hot/they're on fire though/for a while after they're on fire though.

Reply:
Fair for some reason I thought blessed floors needed more then just a fire exe and a water tank.